test/printf.c

```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
```

Comment:
Sounds good, let's leave the test here then. I'll make asm2wasm optimize to create that pattern, and then this will be tested that way.